File: app/src/main/java/databinding/example/com/java/util/BindingConverters.java
package databinding.example.com.java.util;

import android.databinding.BindingConversion;
import android.view.View;

public class BindingConverters {
    @BindingConversion
    public static int booleanToVisibility(boolean isNotVisible){
        return (isNotVisible)? View.GONE : View.VISIBLE;
    }
}

File: app/src/main/java/databinding/example/com/basic/util/BindingConverters.kt
package databinding.example.com.basic.util

import android.databinding.BindingConversion
import android.view.View

object ConverterUtil {
    @JvmStatic fun isZero(number: Int): Boolean {
        return number == 0
    }
}

object BindingConverters{

    @BindingConversion
    @JvmStatic fun booleanToVisibility(isNotVisible: Boolean): Int {
        return if (isNotVisible) View.GONE else View.VISIBLE
    }
}

